server/src/main/java/org/apache/iotdb/db/tools/upgrade/TsFileOnlineUpgradeTool.java

```diff
@@ -152,7 +111,7 @@ private void upgradeFile(List<TsFileResource> upgradedResources)
             if (newChunkGroup) {
```

Comment:
Why do we need to check if it's a new chunk？i.e., Where is the purpose of this variable check?

Reply:
I see this code is not used, I think this is legacy code, I will remove this variable. Thanks.